src/app/Library/CrudPanel/Traits/Search.php

```diff
@@ -390,6 +390,7 @@ public function getEntriesAsJsonForDatatables($entries, $totalRows, $filteredRow
         $rows = [];
 
         foreach ($entries as $row) {
+            $row = $this->setLocaleOnModel($row);
```

Comment:
I spy with my little eye... this PR making changes in other operations than Update.